staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2993,6 +3001,7 @@ type PodSpec struct {
 	// Selector which must match a node's labels for the pod to be scheduled on that node.
 	// More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
 	// +optional
+	// +mapType=atomic
```

Comment:
This is covered by https://github.com/kubernetes/kubernetes/pull/97989/files#diff-5ffbebb40a29f8e491f4ff1e07c618eb3edcc0d5f00a1a71445272d6767d7883R2985 and has included tests. No need to include this in that PR.